cmd/main.go

```diff
@@ -179,7 +179,8 @@ func loadConfigs() (exp.Config, error) {
 			CertPath:    mainflux.Env(envMqttCert, defMqttCert),
 			PrivKeyPath: mainflux.Env(envMqttPrivKey, defMqttPrivKey),
 		}
-		mqttTopic := mainflux.Env(envMqttChannel, defMqttChannel)
+		mqttChannel := mainflux.Env(envMqttChannel, defMqttChannel)
+		mqttTopic := "channels/" + mqttChannel + "/messages"
```

Comment:
Use [`chansPrefix`](https://github.com/mainflux/mainflux/blob/b13a3135ddf0b74290af4dffe532d7d5fe0ae0ce/messaging/nats/pubsub.go#L18) const for this